.nvmrc

```diff
@@ -0,0 +1 @@
+16
```

Comment:
Just curious as to why this is 16 and not 18 like Sensei?

Reply:
Yeah, that sounds convenient. I've switched it to 18 but I had to update the packages. 3c43ac8436c0b36e1ca56d72e1db8a3520c6d8d0 fb45e4a2596c20eb64b4e245d018e2b12352da52